Frameworks/UIKit/UIFont.mm

```diff
@@ -856,102 +321,33 @@ - (instancetype)copyWithZone:(NSZone*)zone {
     return [self retain];
 }
 
-/**
- @Status Interoperable
-*/
-- (void)dealloc {
-    _fileName = nil;
-    _name = nil;
-    _descriptor = nil;
-    [super dealloc];
-}
-
 - (void)_setName:(NSString*)name {
-    _fileName = name;
+    UNIMPLEMENTED();
 }
 
 - (uint32_t)_fontHandle {
-    return (uint32_t)_font;
+    UNIMPLEMENTED();
+    return StubReturn();
 }
 
 - (uint32_t)_sizingFontHandle {
-    return (uint32_t)_sizingFont;
+    UNIMPLEMENTED();
+    return StubReturn();
 }
 
 // Internal methods
 
 // Private message sent from CTFontManager for the implementation of CTFontManagerRegisterGraphicsFont
 - (bool)_CTFontManagerRegisterGraphicsFont:(CGFontRef)font withError:(CFErrorRef*)error {
```

Comment:
Np but we should remember that now some of the public methods are actual STUBs and you have kept track of these then we are good

---
In reply to: [78224805](https://github.com/Microsoft/WinObjC/pull/987#discussion_r78224805) [](ancestors = 78224805,78222243)